features/FEATURE_BLE/targets/TARGET_Cypress/TARGET_CYW43XXX/CyH4TransportDriver.cpp

```diff
@@ -0,0 +1,123 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017-2017 ARM Limited
+ *
```

Comment:
SPDX identifier for new files please

Reply:
Done.
Fixes to PinNamesTypes.h and PortNames.h files from TARGET_Cypress\TARGET_PSOC6 will be merged to the PR on Monday.